Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:bookworm-slim
+
+RUN mkdir -p /opt/fedi-killspam
+VOLUME /config
+WORKDIR /opt/fedi-killspam
+
+
+RUN apt-get update && apt-get install -y curl unzip
+
+ADD . /opt/fedi-killspam/
```

Comment:
`COPY` is more prefered way to add source code

FYI: https://phoenixnap.com/kb/docker-add-vs-copy